src/services-managing.rst

```diff
@@ -164,10 +134,6 @@ Click on :guilabel:`Reset App Server`. A confirmation dialog will popup asking y
 
 Services will contact the ODK-X Cloud Endpoint and attempt to push all configuration and data currently on the tablet up to the specified Cloud Endpoint. A progress dialog will be displayed and, alternatively, the status of resetting the app server can be obtained by looking at the notifications generated by Services in the notification area.
 
-.. note::
```

Comment:
Is this note no longer true? Otherwise it should be retained as a helpful note.

Reply:
Yes, it isn't. Although the user can navigate from that page, they can't really do anything else until the process ends.